src/api/entities/confidential/ConfidentialAsset/index.ts

```diff
@@ -98,6 +98,43 @@ export class ConfidentialAsset extends Entity<UniqueIdentifiers, string> {
     };
   }
 
+  /**
+   * Retrieve all the auditors for this confidential Asset grouped by their type
+   */
+  public async getAuditors(): Promise<GroupedAuditors> {
+    const {
+      id,
+      context: {
+        polymeshApi: {
+          query: { confidentialAsset },
+        },
+      },
+      context,
+    } = this;
+
+    const rawAssetId = serializeConfidentialAssetId(id);
+
+    const assetAuditors = await confidentialAsset.assetAuditors(rawAssetId);
+
+    if (assetAuditors.isNone) {
+      throw new PolymeshError({
+        code: ErrorCode.DataUnavailable,
+        message: 'The Confidential Asset does not exists',
```

Comment:
```suggestion
        message: 'The Confidential Asset does not exists',
        data: { id }
```

(nit) I've been trying to include data more in the errors